android/libraries/rib-base/src/main/java/com/badoo/ribs/android/PermissionRequester.kt

```diff
@@ -0,0 +1,45 @@
+package com.badoo.ribs.android
+
+import com.badoo.ribs.core.Identifiable
+import io.reactivex.Observable
+
+interface PermissionRequester {
+
+    fun checkPermissions(
```

Comment:
Back to code style.
Here it's hard to distinguish `function`'s and `data class`es because of line breaking for functions.

It's quite understandable to do it for classes, but short functions. I don't think so.

Reply:
Fixed.